numpy/_core/strings.py

```diff
@@ -49,9 +49,13 @@
     "rstrip", "strip", "replace", "expandtabs",
 
     # _vec_string - Will gradually become ufuncs as well
-    "mod", "decode", "encode", "center", "ljust", "rjust", "zfill", "upper",
```

Comment:
`center`, `ljust`, `rjust` and `center` should be readded below ufuncs above, which is also the status quo on main. Would you like me to push a fix @charris?

Reply:
> Would you like me to push a fix

Yes, please. Ideally this would follow what happened in main, but there are a couple of missing backports.

Reply:
Turns out I don't have the nessary permissions I think.

```
ERROR: Permission to charris/numpy.git denied to lysnikolaou.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

This diff should do it:

```diff
diff --git a/numpy/_core/strings.py b/numpy/_core/strings.py
index 7e8f93510f..664bc85f80 100644
--- a/numpy/_core/strings.py
+++ b/numpy/_core/strings.py
@@ -49,7 +49,8 @@
     "rstrip", "strip", "replace", "expandtabs",
 
     # _vec_string - Will gradually become ufuncs as well
-    "upper", "lower", "swapcase", "capitalize", "title",
+    "upper", "lower", "swapcase", "capitalize", "title", "center", "ljust",
+    "rjust", "zfill",
 
     # _vec_string - Will probably not become ufuncs
     "mod", "decode", "encode", "translate",
```